ebur128/ebur128.c

```diff
@@ -1457,3 +1457,42 @@ int ebur128_prev_true_peak(ebur128_state* st,
                      st->d->prev_sample_peak[channel_number]);
   return EBUR128_SUCCESS;
 }
+
+void ebur128_reset(ebur128_state *st) {
+  unsigned int i;
```

Comment:
Some of this code is redundant with code in ebur128_init(). Maybe there could be a common function they both call?

Reply:
Fully agree